xmlrunner/runner.py

```diff
@@ -14,13 +14,11 @@ class XMLTestRunner(TextTestRunner):
     """
     A test runner class that outputs the results in JUnit like XML files.
     """
-    def __init__(self, output='.', outsuffix=None, stream=sys.stderr,
-                 descriptions=True, verbosity=1, elapsed_times=True,
-                 failfast=False, buffer=False, encoding=UTF8,
-                 resultclass=None):
-        TextTestRunner.__init__(self, stream, descriptions, verbosity,
-                                failfast=failfast, buffer=buffer)
-        self.verbosity = verbosity
+    def __init__(self, output='.', outsuffix=None, 
+                 elapsed_times=True, encoding=UTF8,
+                 resultclass=None,
+                 **kwargs):
+        super(XMLTestRunner, self).__init__(**kwargs)
```

Comment:
The issue is caused for that `XMLTestRunner` doesn't support `warnings` parameter that is added to `TextTestRunner` in python3: https://github.com/python/cpython/blob/v3.4.8/Lib/unittest/runner.py#L129

This change receives any unknown parameters and pass them through to `TextTestRunner`.
This ensures `XMLTestRunner.__init__` is always compatible with `TextTestRunner.__init__` and clarifies that `XMLTestRunner` itself doesn't handle those parameters.